packer/render/squidex/squidex.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/sh
 
-echo "Waiting 60 seconds ..."
+echo "Waiting 60 seconds for MongoDB ..."
 sleep 60
 
+if [ "x$DOMAIN" = "x" ]; then
+    export DOMAIN=$RENDER_EXTERNAL_HOSTNAME
+fi
+
 export URLS__BASEURL="https://${DOMAIN}"
 export EVENTSTORE__MONGODB__CONFIGURATION="mongodb://${MONGO}"
 export STORE__MONGODB__CONFIGURATION="mongodb://${MONGO}"
 
+export IDENTITY__GOOGLECLIENT=""
```

Comment:
I think we have to do it as well for all other cloud providers like GITHUB and MICROSOFT